packages/promotion/src/migrations/Migration20231218142613.ts

```diff
@@ -0,0 +1,9 @@
+import { Migration } from '@mikro-orm/migrations';
```

Comment:
suggestion: Going forward, instead of adding new migrations when changing schemas, I would suggest we ship one single file for the first official release

Reply:
Yes, agreed. this is just a placeholder for now